econml/inference.py

```diff
@@ -689,6 +701,8 @@ def population_summary(self, alpha=0.1, value=0, decimals=3, tol=0.001, output_n
             The population summary results instance contains the different summary analysis of point estimate
             for sample X on each treatment and outcome.
         """
+        treatment_name = self.treatment_name if treatment_name is None else treatment_name
+        output_name = self.output_name if output_name is None else output_name
```

Comment:
```suggestion
        treatment_name = treatment_name or self.treatment_name
        output_name = output_name or self.output_name
```

Reply:
The changes are out of date, but will address in a separate commit.